docs/redoc/master/openapi.json

```diff
@@ -11970,6 +12065,138 @@
             }
           }
         }
+      },
+      "QueryGroupsRequest": {
+        "type": "object",
+        "required": [
+          "group_by",
+          "group_size",
+          "limit"
```

Comment:
Let's keep the required params minimal., in this case only require to set `group_by`. Some reasonable defaults would be limit=10 and group_size=3, wdyt?